internal/controller/utils/reconciler.go

```diff
@@ -79,6 +79,10 @@ func (r ReconcileStepResult) HasErrors() bool {
 	return len(r.errs) > 0
 }
 
+func (r ReconcileStepResult) NeedsRequeue() bool {
```

Comment:
Please add a comment for this public function to fix the linting error

Reply:
done